### ISSUE TITLE
Create user help documentation

### ISSUE THREAD

**Issue by brynwhyman** [kinglozzer/silverstripe-session-manager#15]
~**Blocked until https://github.com/silverstripe/silverstripe-framework/issues/9883 is merged** - we need finalised UI before we can describe it and take screenshots~

### Overview
The new designs include a 'Learn more' link in the CMS UI.
Some [user help documentation](https://userhelp.silverstripe.org/en/4/optional_features/managing_devices/) needs to be written and the button in the CMS created.

### Acceptance Criteria
- [ ] Information covering how the feature is intended to work and how users interact with it is documented
- [ ] The user help documentation is accessible via the CMS UI (through the 'Learn more' button)
  - [ ] Update [the help link in MemberExtension](https://github.com/silverstripe/silverstripe-session-manager/blob/1/src/Extension/MemberExtension.php)
- [ ] Server requirements are updated to include information on this module and include gotchas (from the caveats section of the module docs): https://docs.silverstripe.org/en/4/getting_started/server_requirements/
- [ ] The "[Logging in](https://userhelp.silverstripe.org/en/4/managing_your_website/logging_in/#logging-in)" section of User Help has received relevant updates according to Session Manager and Login Forms changes (see notes below).

### Notes
- The user documentation should be stored in this module and made available for viewing under the 'optional features' section in the CMS user help docs.
- There is an internal guide on how to write User Help on Confluence (ask Paul)
- The base login user help could do with some updates for how Login Forms and non Login Forms may appear. For example:
  - Updated screenshot for demo login 
  - Additional screenshot with Login Forms experience (to highlight the contrast of experiences)
  - Link from the "[Keep me signed in](https://userhelp.silverstripe.org/en/4/managing_your_website/logging_in/#staying-signed-in-remember-me-functionality)" segment to the session manager guides for more information. Session manager docs will probably move to sit next to "Log in" guides once it is in core.
  - Add text at the bottom of the page "If you’re having trouble accessing your account please contact the website owner".

### PRs / Actions
- [x] https://github.com/silverstripe/silverstripe-session-manager/pull/36 (TODO: update link to userhelp)
- [x] https://github.com/silverstripe/silverstripe-session-manager/pull/78
- [x] https://github.com/silverstripe/doc.silverstripe.org/pull/232
- [x] Add `NETLIFY_BUILD_HOOK` secret to session-manager repo (ask @bergice)
- [ ] Apply relevant updates to "Loggin in" section of User Help
- [x] Update login form image https://github.com/silverstripe/silverstripe-userhelp-content/pull/163
- [ ] Follow up on updating the login userhelp page to refer to session-manager userdocs when it is stable.
- [x] https://github.com/silverstripe/silverstripe-session-manager/pull/81
- [x] https://github.com/silverstripe/silverstripe-session-manager/pull/84

**Comment by brynwhyman** [comment]
> The user documentation should be stored in this module and made available for viewing under the 'optional features' section in the CMS user help docs.

I can't find docs on how to get the docs appearing in userhelp.silverstripe.org, but here's at least one thing that's required: https://github.com/silverstripe/silverstripe-mfa/commit/03e70007a577ef56c4b2e97bd573437d83585db3

**Comment by maxime-rainville** [comment]
We're changing the link to https://userhelp.silverstripe.org/en/4/optional_features/managing_devices for now. https://github.com/silverstripe/silverstripe-session-manager/pull/62/files#r613873046

Might need to change to something else later on.

**Comment by bergice** [comment]
> Server requirements are updated to include information on this module and include gotchas (from the caveats section of the module docs): https://docs.silverstripe.org/en/4/getting_started/server_requirements/

I don't get this.
This is an optional module, and it's already documented on the module itself.

**Comment by clarkepaul** [comment]
I've just gone through and done a review of the user help guides, hit me up if you have any questions or problems with it.

**Comment by brynwhyman** [comment]
>Link from the "Keep me signed in" segment to the session manager guides for more information. Session manager docs will probably move to sit next to "Log in" guides once it is in core.

Thinking about it a bit, I do feel like it could be too soon to make reference to session-manager in the core 'logging in' userhelp? The module is yet to go stable so it feels like it might just confuse people if they were to read it but don't have the feature?

When we move session manager to core, we'll definitely be updating the userhelp then.

In the meantime though, while we're hear I think the screenshot in https://userhelp.silverstripe.org/en/4/managing_your_website/logging_in/ should be replaced to one with login-forms - that's been the default for multiple minor releases now.

**Comment by clarkepaul** [comment]
Added another suggestion by Matt P
> Add text at the bottom of the page "If you’re having trouble accessing your account please contact the website owner as they can reset your account".

**Comment by bergice** [comment]
> Added another suggestion by Matt P
> 
> > Add text at the bottom of the page "If you’re having trouble accessing your account please contact the website owner as they can reset your account".

There's already a https://userhelp.silverstripe.org/en/4/managing_your_website/logging_in/#lost-password section.

**Comment by clarkepaul** [comment]
~~The use case was more for non CMS users using the login page (they wont realise its the login screen for the CMS itself) and viewing CMS help will be confusing and potentially alarming.~~ For example lost password doesn't make a difference if your email address isn't accepted, you need to contact the site owner.

~~We've had a few follow on convos yesterday that we think we are going to need to have an option to remove the "Need help?" link and Silverstripe CMS logo if the login is being used by website users (separate issue), or have those things disabled by default and can be enabled if only used by CMS users.~~

@bergice We are going to remove the "Need help?" link now so we don't have to put that "contact your site owner" text in user help anymore, but on the flip side also doesn't do much harm especially if an email address isn't accepted.

**Comment by clarkepaul** [comment]
I'm happy with the docs, everything resolved from what I can tell.

**Comment by bergice** [comment]
Closing.

Opened https://github.com/silverstripe/silverstripe-session-manager/issues/85
